frontend/src/metabase/admin/settings/app/components/SettingsEditor/SettingsEditor.jsx

```diff
@@ -171,32 +171,23 @@ class SettingsEditor extends Component {
           saveStatusRef={this.saveStatusRef}
           elements={activeSection.settings}
           settingValues={settingValues}
-          updateSetting={this.updateSetting}
+          derivedSettingValues={derivedSettingValues}
+          updateSetting={this.updateSetting.bind(this)}
+          onChangeSetting={this.handleChangeSetting.bind(this)}
+          reloadSettings={this.props.reloadSettings}
         />
       );
-    } else {
-      return (
```

Comment:
This JSX block got moved to `SettingsSection` so that it can be reused in `SettingsEmailForm`